src/components/shared-components/rating/starRating.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react";
+import Star from "../../../assets/images/filled.png";
+import EmptyStar from "../../../assets/images/empty.png";
+
+class Rating extends Component {
+  render() {
+    let ratingObj = this.props.ratingObj;
+    let stars = [];
+
+    if(ratingObj < 0 || ratingObj >5){
```

Comment:
Nice but looks like that all this block of code could move it to another function and using the state for stars instead of declaring a let variable in the render function